modules/users/client/directives/password-verify.client.directive.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+angular.module('users')
+  .directive("passwordVerify", function() {
+    return {
+      require: "ngModel",
```

Comment:
@jloveland JSHint is complaining about not using single quotes here. Could you fix and send another PL?